File: src/main/java/no/nav/pensjon/regler/domain/beregning/penobjekter/Sykepenger.java
package no.nav.pensjon.regler.domain.beregning.penobjekter;

import no.nav.pensjon.regler.domain.kode.YtelsekomponentTypeCti;

/**
 * Created by N123422 on 07.11.2014.
 */
public class Sykepenger extends MotregningYtelseskomponent {

    private static final long serialVersionUID = 715401119582214508L;

    public Sykepenger() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("SP");
    }

    public Sykepenger(Sykepenger sykepenger) {
        super(sykepenger);
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning/penobjekter/Sykepenger.kt
package no.nav.pensjon.regler.domain.beregning.penobjekter

import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

/**
 * Created by N123422 on 07.11.2014.
 */
class Sykepenger : MotregningYtelseskomponent() {
    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.SP
}